src/diffusers/modeling_utils.py

```diff
@@ -20,6 +20,7 @@
 import torch
 from torch import Tensor, device
 
+import accelerate
```

Comment:
We should not make `accelerate` a hard dependency here . Could you wrap it into a:

```
if accelerate_is_available():
    import accelerate
else:
    raise ImportError("Please install accelerate via `pip install accelerate`")
```

below the `if device_map == "auto"` method?

Reply:
@patrickvonplaten just did it, thank you for the suggestion.